test/multimanifestmodule.test.ts

```diff
@@ -128,35 +96,38 @@ suite('multimanifest module', () => {
     });
     */
 
-    test('triggerManifestWs should return error', () => {
+    test('triggerManifestWs should return error', async() => {
         let stubAuthorize_f8_analytics =  sandbox.stub(authextension, 'authorize_f8_analytics').yields(null);
-        let promiseTriggerManifestWs = multimanifestmodule.triggerManifestWs(context, 'pom.xml', provider, previewUri);
-        promiseTriggerManifestWs.catch((err)=>{
-            expect(err).equals(undefined);
-        });
+        try {
+            await multimanifestmodule.triggerManifestWs(context, provider, previewUri);
+        } catch (err) {
+            expect(err).equals('Unable to authenticate.');    
```

Comment:
and can you move this error message outside the catch block;
eg:
```
            let savedErr;
            try {
                await multimanifestmodule.triggerManifestWs(context, provider, previewUri);
            } catch (err) {
                savedErr = err;
            }
            expect(savedErr).equals('Unable to authenticate.');  

```

Reply:
any benefit with this?

Reply:
yes if it fails then only we will get the error message otherwise it will fail the test.